content/en/docs/overview/variables.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Variables"
+weight: 210
+---
+
+The short assignment statement `:=` declares a variable and assigns a value to it.
+The type of variable is inferred from the value (type inference).
+```golang
+package main
+
+import "fmt"
+
+func main() {
+	name := "foo.txt"
```

Comment:
Maybe add the type as a comment to make it extra clear what is happening? This could be confusing though, if they do not realize it is a comment...
```suggestion
	name := "foo.txt"  // string
```

Reply:
It needs to be clear, that every variable receives a type that cannot be changed later on. It is the same as writing:
```
var name string
string = "foo"

string := "foo"
```

Reply:
tried to explain what type the variables get with comments. 

at this point we can't really say that these two ways are the same because we did not introduce the `var`